### PR TITLE
Typo, grammar, and continuity fixes

### PR DESCRIPTION
**Pull request by timmc**
Suggestions for things I noticed while reading through the draft.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -377,7 +377,7 @@ <h2>Capabilities Are Safer</h2>
             if an attacker gets you to open a message in your email client
             which contains an image which triggers an arbitrary code execution
             vulnerability in the image library it is using, an attacker can
-            use this to do everything that you can: read or destroy your
+            use this to do everything that you can: they can read or destroy your
```
Comment (timmc) [inline index.html:380]: Or remove "they can" from each item in the list, either way.

**Review by msporny — APPROVED** [review]

**Comment by msporny** [comment]
Thanks a ton for the fixes @timmc! :)

Just as a warning, the concepts in the spec are stable, but the examples/content is horribly out of date. It's not tracking the implementations very closely. If you're planning to use any of this, latest code can be found here:

https://github.com/digitalbazaar/http-signature-zcap-invoke

and here:

https://github.com/digitalbazaar/http-signature-zcap-verify

**Comment by timmc** [comment]
Cool, good to know!

**Comment by dlongley** [comment]
This is also vital (the above libraries use it): https://github.com/digitalbazaar/ocapld.js

**Pull request merged**